Games/18 Wheels of Steel Across America/18WoSAA/script.js

```diff
@@ -0,0 +1,10 @@
+include(["Functions", "QuickScript", "LocalInstallerScript"]);
+
+new LocalInstallerScript()
+    .name("18 Wheels of Steel: Across America")     // name of the application
```

Comment:
Please remove the comments here (I only used them in the wiki as an explanation).